### PR TITLE
[AIRFLOW-6753] Run REST API tests when DAGs are serialized

### PR DESCRIPTION
**Pull request by anitakar**
---
Issue link: [AIRFLOW-6753](https://issues.apache.org/jira/browse/AIRFLOW-6753)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: tests/www/api/experimental/test_endpoints.py [hunk 1]
```diff
@@ -80,239 +90,265 @@ def test_info(self):
         self.assertEqual(version, resp['version'])
 
     def test_task_info(self):
-        url_template = '/api/experimental/dags/{}/tasks/{}'
+        with conf_vars(
+            {("core", "store_serialized_dags"): self.dag_serialzation}
+        ):
+            url_template = '/api/experimental/dags/{}/tasks/{}'
```
Comment (turbaszek) [inline tests/www/api/experimental/test_endpoints.py:96]: I think using `conf_vars` as decorator will reduce number of code changes :)

Comment (mik-laj) [reply tests/www/api/experimental/test_endpoints.py:96]: This will not work properly with`parameterized_class`

**Comment by mik-laj** [comment]
Travis is sad. Can you take care of him?

**Review by mik-laj — APPROVED** [review]

**Comment by anitakar** [comment]
Connected to another ticket https://issues.apache.org/jira/browse/AIRFLOW-6753

**Pull request merged**